### PR TITLE
Allow to optionally skip shading of the blob

### PR DESCRIPTION
**Pull request by kotarak**

### CODE REVIEW

File: src/unrepl/make_blob.clj [hunk 1]
```diff
@@ -56,36 +56,58 @@
...
-                       :except [#"clojure\..*" 'unrepl.core]})
-        code (-> template
-               (str/replace "unrepl.repl"
-                 (str (shaded-libs 'unrepl.repl)))
-               (str/replace "<BLOB-PAYLOAD>" (str shaded-code-sb)))]
+        code     (if shade?
+                   (let [shaded-code-sb (StringBuilder.)
+                         shaded-libs (shade/shade 'unrepl.repl
+                                                  {:writer (fn [_ ^String code] (.append shaded-code-sb code))
+                                                   :except [#"clojure\..*" 'unrepl.core]})]
```
Comment (cgrand) [inline src/unrepl/make_blob.clj]: Shouldn't `:except #".*"` prevent shading?

Comment (cgrand) [reply src/unrepl/make_blob.clj]: ok I think https://github.com/Unrepl/unrepl/commit/41eea93c1e4eb372169ccd7aa5ed191c362c8046 is relevant

Comment (kotarak) [reply src/unrepl/make_blob.clj]: I will have a look later today. Now `:except #".*"` will work. However it feels very ugly to run this whole non-trivial machinery to do … nothing. I'd rather have shade doesn't read anything, but only feed it the list a relevant things.

Comment (kotarak) [reply src/unrepl/make_blob.clj]: As a sidenote: the `--noshade` option should probably be something like `--except`.

**Pull request merged**